src/main/java/com/monumental/controllers/MonumentController.java

```diff
@@ -92,6 +128,35 @@ public Monument updateMonument(@PathVariable("id") Integer id, @RequestBody Upda
         return this.monumentService.updateMonument(id, newMonument);
     }
 
+    private static class ToggleIsActiveRequest {
+        public boolean isActive;
+    }
+
+    /**
+     * Change the value of isActive on a Monument
+     * @param id - ID of the Monument to update
+     * @param request - ToggleIsActiveRequest containing the new value for isActive
+     * @return Monument - The updated Monument
+     */
+    @PutMapping("/api/monument/active/{id}")
+    @PreAuthorize(Authorization.isResearcherOrAbove)
+    public Monument updateMonument(@PathVariable("id") Integer id, @RequestBody ToggleIsActiveRequest request) {
```

Comment:
Maybe we should rename this to `updateMonumentIsActive`?

Also, what's the different between `@RequestBody` and `@ModelAttribute`?

Reply:
https://stackoverflow.com/a/31990142/10044594

Reply:
I thought `@RequestParam` was for query params